libafl_targets/src/cmps/mod.rs

```diff
@@ -535,18 +555,16 @@ impl CmpMap for AFLppCmpLogMap {
                         self.vals.operands[idx][execution].v0,
                         self.vals.operands[idx][execution].v1,
                     ))),
-                    // TODO handle 128 bits cmps
+                    // TODO handle 128 bits & 256 bits cmps
                     // other => panic!("Invalid CmpLog shape {}", other),
                     _ => None,
                 }
             }
         } else {
             unsafe {
-                let v0_len = self.vals.fn_operands[idx][execution].v0_len & (0x80 - 1);
```

Comment:
why you don't look at len anymore?

Reply:
From my understanding, this was only necessary due to the length of the v1/v0 array being 31 instead of 32 right?

Reply:
hmm no. 
sometimes each fn operands has its length (not necessarily 32). so you should keep the length

Reply:
Also, random question, could we get around the allocation here?

Reply:
not possible.
the tokens have to be copied from the shared mem. else it'll be overwritten in the next round

Reply:
But they could be copied to a preexisting buffer

Reply:
but we don't know how many of them needs copying so we can't decide its size.

Reply:
Like, just memcpy the existing struct and make a nice accessor around it that returns a slice

Reply:
See #2442